test/e2e/e2e_suite_test.go

```diff
@@ -65,6 +85,16 @@ var (
 
 	// skipCleanup prevents cleanup of test resources e.g. for debug purposes.
 	skipCleanup bool
+
+	// defines how CAPV should behave during this test.
+	testMode string
+
+	// defines which type of infrastructure this test targets.
+	testTarget string
+
+	// byoConfigPath is the path to the e2e config file containing overrides for a tests targeting a user provided vCenter instance.
+	// those overrides will be applied on top of the content for configPath (only variables and intervals are considered).
+	byoConfigPath string
```

Comment:
I wonder if this also makes sense for the other testTargets (Just to be able to override stuff in general).

We could call it overridesConfigPath and use it whenever it is set